music21/analysis/neoRiemannian.py

```diff
@@ -29,12 +31,20 @@
 class LRPException(exceptions21.Music21Exception):
     pass
 
-def L(c, raiseException=False):
+def simplerEnharmonics(self):
```

Comment:
This isn't in a class so the variable should be called "ch" or something like that, not "self".